docs/cluster-configuration.md

```diff
@@ -304,11 +304,33 @@ metadata:
       [
         {
           "op": "add",
-          "path": "/configuration/migrations",
-          "value": '{"allowPostCopy": "true"}'
+          "path": "/spec/configuration/migrations",
+          "value": {"allowPostCopy": true}
         }
       ]
 ```
+
+From CLI it will be:
+```bash
```

Comment:
i would suggest to format each line separately, because the lines are very long, which cause the reader to scroll horizontally back and forth while reading this section.